config/environments/production.rb

```diff
@@ -58,6 +58,10 @@
   config.gias_api_user = ENV["GIAS_API_USER"]
   config.gias_api_password = Rails.application.credentials.GIAS_API_PASSWORD
 
+  config.dqt_client_api_key = Rails.application.credentials.DQT_CLIENT_API_KEY
```

Comment:
Could do this in application.rb:

```
config.dqt_client_api_key = Rails.application.credentials.DQT_CLIENT_API_KEY || ENV["DQT_CLIENT_API_KEY"]
```

and so on.

Reply:
I've just copy pasted the current convention from `deployed_development`. Those working more closely on ECF may have an opinion on how they want it.